package.json

```diff
@@ -36,11 +36,15 @@
     "e2e-format": "yarn workspace e2e format",
     "cli-lint": "yarn workspace @lightdash/cli lint",
     "cli-format": "yarn workspace @lightdash/cli format",
-    "lint": "run-s common-lint backend-lint frontend-lint e2e-lint cli-lint",
-    "format": "run-s common-format backend-format frontend-format e2e-format cli-format",
-    "test": "run-s backend-test",
+    "warehouses-lint": "yarn workspace @lightdash/warehouses lint",
+    "warehouses-format": "yarn workspace @lightdash/warehouses format",
+    "warehouses-test": "yarn workspace @lightdash/warehouses test",
+    "warehouses-build": "yarn workspace @lightdash/warehouses build",
+    "lint": "run-s common-lint backend-lint frontend-lint e2e-lint cli-lint warehouses-lint",
+    "format": "run-s common-format backend-format frontend-format e2e-format cli-format warehouses-format",
+    "test": "run-s backend-test warehouses-test",
     "dev": "npm-run-all common-build --parallel backend-dev frontend-dev",
```

Comment:
should we build the warehouse package here ? like we do it for common

Reply:
Yes! Thanks! Also the order of the builds was wrong thanks

Reply:
The order is not wrong for dev since FE and BE run as separate servers. They can be started with any order.